### PR TITLE
Release 2.2.0-alpha

### PR DESCRIPTION
**Pull request by linuxwolf**
Release **Lockwise** 2.2.0-alpha

### CODE REVIEW

**Comment by linuxwolf** [comment]
**NOTE** Holding off a bit to see if we can sneak in #232

File: docs/release-notes.md [hunk 1]
```diff
@@ -1,4 +1,30 @@
-# Firefox Lockbox for Desktop Release Notes
+# Firefox Lockwise for Desktop Release Notes
+
+## 2.2.0-alpha
+
+_Date: 2019-05-10_
+
+_**Firefox** Lockbox_ is now _**Firefox** Lockwise_! With the new name also comes a new look!
```
Comment (devinreams) [inline docs/release-notes.md]: ❗️ 😍 ❗️😍 ❗️

**Comment by linuxwolf** [comment]
Awaiting an approval from @mozilla-lockwise/product

**Comment by sandysage** [comment]
Ship it!

**Pull request merged**